src/java/voldemort/server/VoldemortConfig.java

```diff
@@ -355,6 +356,8 @@ public VoldemortConfig(Props props) {
                                                      VoldemortConfig.DEFAULT_KERBEROS_REALM);
         this.fileFetcherClass = props.getString("file.fetcher.class",
                                                 VoldemortConfig.DEFAULT_FILE_FETCHER_CLASS);
+        this.readOnlyStatsFileEnabled = props.getBoolean("readonly.stats.file.enabled", true);
+        this.readOnlyMaxVersionsStatsFile = props.getInt("readonly.stats.file.max.versions", 50);
```

Comment:
For a directory with 5 files it is about 350 bytes + 500 bytes for header/footer. If the directory has 150 files it is going to 10KB. If we keep 50 stats files that is going to 500 MB per store. For a cluster with 150 stores that is already 75 GB and which is 7.5% on some of our production clusters. So I am worried about increasing this number beyond 50. Please let me know if this number makes sense ?